packages/ui/v2/core/apps/AppCard.tsx

```diff
@@ -0,0 +1,131 @@
+import type { Credential } from "@prisma/client";
+
+import useAddAppMutation from "@calcom/app-store/_utils/useAddAppMutation";
+import { InstallAppButton } from "@calcom/app-store/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import showToast from "@calcom/lib/notification";
+import { trpc } from "@calcom/trpc/react";
+import { App } from "@calcom/types/App";
+import { Icon } from "@calcom/ui/Icon";
+import Button from "@calcom/ui/v2/core/Button";
+
+interface AppCardProps {
+  app: App;
+  credentials?: Credential[] | undefined;
+}
+
+export default function AppCard({ app, credentials }: AppCardProps) {
+  const { t } = useLocale();
+  const { data: user } = trpc.useQuery(["viewer.me"]);
+
+  const mutation = useAddAppMutation(null, {
+    onSuccess: () => {
+      showToast(t("app_successfully_installed"), "success");
+    },
+    onError: (error) => {
+      if (error instanceof Error) showToast(error.message || t("app_could_not_be_installed"), "error");
+    },
+  });
+
+  const allowedMultipleInstalls = app.categories && app.categories.indexOf("calendar") > -1;
+  const appAdded = (credentials && credentials.length) || 0;
+
+  return (
+    <div
+      className="relative flex h-64 flex-col rounded-md border border-gray-300 p-5"
+      data-testid={`app-store-app-card-${app.slug}`}>
+      <div className="flex">
+        {
+          // eslint-disable-next-line @next/next/no-img-element
```

Comment:
Should this be turned off in the top level eslint settings? Or why aren't you using the Next.js Image component?

Reply:
[This is already turned off](https://github.com/calcom/cal.com/blob/015ed6d4f9969351b8676c41dc5da5cff7159b75/packages/config/eslint-preset.js#L16) so it's redundant